backend/mora/service/handlers.py

```diff
@@ -163,18 +138,20 @@ def prepare_terminate(self, request: dict):
         """
         raise NotImplementedError
 
-    @abc.abstractmethod
-    def submit(self) -> str:
+    def submit(self, result=None) -> str:
```

Comment:
You've changed the signature of the base method, without changing any of the inherited ones. I suggest removing this new parameter, and just saving the result on the object instead.